tests/pkg/fixtures/application_queries.go

```diff
@@ -244,23 +244,12 @@ func GenerateOneTimeTokenForApplicationWithSuggestedToken(t require.TestingT, ct
 	return oneTimeToken
 }
 
-func UnassignApplicationFromScenarios(t require.TestingT, ctx context.Context, gqlClient *gcli.Client, tenantID, applicationID string, defaultScenarioEnabled bool) {
+func UnassignApplicationFromScenarios(t require.TestingT, ctx context.Context, gqlClient *gcli.Client, tenantID, applicationID string) {
 	labelKey := "scenarios"
-	if defaultScenarioEnabled {
-		defaultValue := "DEFAULT"
 
-		scenarios := []string{defaultValue}
-		var labelValue interface{} = scenarios
-
-		setLabelRequest := FixSetApplicationLabelRequest(applicationID, labelKey, labelValue)
-		label := graphql.Label{}
-		err := testctx.Tc.RunOperationWithCustomTenant(ctx, gqlClient, tenantID, setLabelRequest, &label)
+	deleteLabelRequest := FixDeleteApplicationLabelRequest(applicationID, labelKey)
```

Comment:
Why do we delete the labels instead of calling unassign from formation?

Reply:
Because the deletion of the application label(if the label is == `scenarios`) leads to unassignment of all formations. Reference - https://github.com/kyma-incubator/compass/blob/060b16d3fecf6315ffcf04a833b81360701b3283/components/director/internal/domain/application/service.go#L799.